provider/postgis/postgis.go

```diff
@@ -201,38 +207,43 @@ func BuildDBConfig(cs string) (*pgxpool.Config, error) {
 		"application_name":              "tegola",
 	}
 
-	var hasInit bool
+	var hstore hstoreOID
```

Comment:
If we persist this value outside of the AfterConnect call, then it will only be set on the first connection. If I understand you correctly, the hstore registration needs to happen for every connection. If that's the case, then we should move this var to inside the AfterConnect function.

Reply:
I'm not sure I understand your concern.
The idea is that I only query the database once (for the first connection) for the OID of hstore, and save the OID value per provider in the hstore var. 
Now on the next connection the query is skipped, however the datatype is registered with the OID of the hstore var.
If I were to add a log line into the `if !hstore.hasInit` I'd only see it once, yet registering the datatype happens over and over again.